### PR TITLE
Manuscripts and Special Collections visit

### PR DESCRIPTION
**Pull request by GingertronMk1**

### CODE REVIEW

**Comment by nhyty** [comment]
We got to 16-30152m_ACC2699

and have now given up for the day, can now be reviewed and merged

**Comment by samozzy** [comment]
@nhyty Excellent work! 32 new shows and almost 300(!!!) new people!

**Comment by BenWoodford76** [comment]
@samozzy Don’t forget @GingertronMk1

**Comment by samozzy** [comment]
@BenWoodford76 Blame how Telegram replies to comments 😝

File: _shows/72_73/beckett_festival_lunch.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: A Beckett Festival - Lunchtimes
+season: In House
+period: Autumn
+playwright: Samuel Beckett
+season_sort: 130
```
Comment (samozzy) [inline _shows/72_73/beckett_festival_lunch.md]: My reading of this is that Programme C occurs in the evenings of the days that Lunchtimes occur. Therefore the first performance of Lunchtimes is before that of PC, so can the season sorts be swapped?

**Comment by wjdp** [comment]
Fucking amazing, great job!

Guessing these were from programmes, have we got photos of these?

File: _shows/72_73/tis_pity_shes_a_whore.md [hunk 2]
```diff
@@ -3,12 +3,84 @@ title: "'Tis Pity She's A Whore"
...
+- role: Organist
+  name: Derek Carden
+- role: Fights
+  name: John Ironmonger
+- role: Front-Of-House Manger
+  name: Janet Bell
+
 ---
 
 Tis Pity She's a Whore is a tragedy written by John Ford. It was likely first performed between 1629 and 1633.
```
Comment (samozzy) [inline _shows/72_73/tis_pity_shes_a_whore.md]: The title of the show should be in single quotes, and is missing an apostrophe before 'Tis'.

File: _shows/73_74/a_resounding_tinkle.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: A Resounding Tinkle
+playwright: N F Simpson
```
Comment (samozzy) [inline _shows/73_74/a_resounding_tinkle.md]: We have this playwright for [One Way Pendulum](https://history.newtheatre.org.uk/years/65_66/one_way_pendulum/) but without a space between the N and F, should be corrected.

**Comment by GingertronMk1** [comment]
@wjdp Weird situation with that, we can't just have the files but we can copy all the info from them, else we'd probably have the files uploaded

File: _shows/73_74/alberts_bridge.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: Alberts' Bridge
```
Comment (samozzy) [inline _shows/73_74/alberts_bridge.md]: Apostrophe should be before the s.

**Comment by nhyty** [comment]
@GingertronMk1 yeah, there is also like another 2 or 3 sessions of info to go, we ran out of time and will power today.

File: _shows/73_74/oedipus_rex.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+title: Oedipus Rex
+playwright: Sophocles
+translator: John McRae
+season: In House
+period: Spring
+season_sort: 250
+date_start: 1974-016-24
```
Comment (samozzy) [inline _shows/73_74/oedipus_rex.md]: Erroneous 0

File: _shows/73_74/ryders_to_the_sea.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+date_end: 1974-07-02
+season_sort: 210
+venue: New Theatre
+
+crew:
+- role: Director
+  name: Crispin Harris
+---
+
+Synge was probably the greatest of the dramatists associated with the Abbey Theatre, Dublin. By living among the fisherman of the Arran Islands, he absorbed Irish Folklore and gave it dramatic form. 'Riders to the Sea' a "tragedy of classical simplicity" set on one such island in a time of poverty ad great sorrow.
```
Comment (samozzy) [inline _shows/73_74/ryders_to_the_sea.md]: Should be 'poverty **and** great sorrow'.

File: _shows/73_74/the_stronger.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+period: Spring
+season_sort: 250
+
+crew:
+- role: Director
+  name: John McRae
+
+---
+
+Two actresses meet by chance on Christmas Eve n a cafe. One is married, the other is not. One talks, the other remains silent, but between the two a story unfolds of past conquests and present victories. Of the two, the silent one and the chatterer, who is the stronger?
```
Comment (samozzy) [inline _shows/73_74/the_stronger.md]: 'Christmas Eve **in** a cafe'

File: _shows/74_75/icarus_mother.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+- role: Jill
+  name: Theresa Norris
+- role: Pat
+  name: Lorna Tanklovitch
+- role: Frank
+  name: Kev Buxton
+- role: Howard
+  name: Vinny Cleary
+
+crew:
```
Comment (samozzy) [inline _shows/74_75/icarus_mother.md]: This has the same crew as Chicago. Is this correct?

File: _shows/74_75/killers.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+---
+title: Killer
+playwright: Kevin Buxton
+student_written: true
+season: In-House
```
Comment (samozzy) [inline _shows/74_75/killers.md]: No hyphen please

File: _shows/71_72/cal.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+crew:
+- role: Director
+  name: Richard Baker
+
+---
+
+> "Treat us as though we were children... Nice children, of course, children that you're fond of, but not as gorwn-ups. And for God's sake, whatever you do, don't praise us - that drives us mad"
+
+- Arthur Shields to Frank O'Connor, his advice to O'Connor who had just been elected to the boardroom of the Abbey Theatre.
```
[the remark this answers is not in the document]
Comment (samozzy) [reply _shows/71_72/cal.md]: Others like this have `\-` to escape bulleting. Should this be the same?

**Comment by wjdp** [comment]
@GingertronMk1 They're all digital anyway? Stupid

File: _shows/71_72/cal.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (nhyty) [reply _shows/71_72/cal.md]: shall fix  then

File: _shows/71_72/festival_of_one_act_plays.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Festival of One Act Plays
+playwright: Various
```
Comment (samozzy) [inline _shows/71_72/festival_of_one_act_plays.md]: Must be lowercase

**Pull request merged**